java/registry/src/main/java/io/opensaber/registry/helper/RegistryHelper.java

```diff
@@ -63,5 +76,41 @@ public String addEntity(JsonNode inputJson, String userId) throws Exception {
         }
         return recordId.toString();
     }
+
+    /**
+     * Get entity details from the DB and modifies data according to view template
+     * @param inputJson
+     * @param requireLDResponse
+     * @return
+     * @throws Exception
+     */
+    public JsonNode readEntity(JsonNode inputJson, boolean requireLDResponse) throws Exception {
```

Comment:
Create another overloaded method that takes only one param...Let user-service use that.

Reply:
Created new method which can only access by user-service

Reply:
Isn't this silly to repeat. Why is the new method created not calling the readEntity(inputJson, false)?

Reply:
My apologies for this weird changes, changed accordingly.